### PR TITLE
fix: fix the problem that the Get method is incorrect when there is a…

### PR DESCRIPTION
**Pull request by xcrossed**
… hashmap conflict

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea/
 .vscode/
+.history/
```
Comment (tjgurwara99) [inline .gitignore:3]: Why was this added? I don't think the repository needs this, you can keep it in your local and not commit it to the repository.

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]
Please provide a description explaining why the change was made. From the outset, looking at the tests, I can't see why this change was necessary. Please provide a description and a test case where you expect the previous implementation to fail and then we can review this further 😄

File: .gitignore (hunk 1 above)
Comment (xcrossed) [reply .gitignore:3]: this dir  is a vscode extension local history use This is a directory used by a vscode or idea plugin called local history, which I can remove.

**Comment by xcrossed** [comment]
> 
The scenario where this happens is: when there is a hash key conflict, the hashmap implementation at that time stores the conflicting nodes through a linked list. Therefore, when the Get method is executed, the linked list needs to be traversed.
